docs/docs/guides/troubleshooting.mdx

```diff
@@ -139,6 +139,22 @@ See [Multiple versions of Reanimated were detected](#multiple-versions-of-reanim
 **Solution:** Please upgrade to a newer version of React Native or downgrade to an older version of Reanimated.
 See [the compatibility table](compatibility) for a full list of supported versions of React Native.
 
+## Warnings
+
+### Reduced motion setting is enabled on this device.
+
+**Problem:** This warning is displayed to avoid confusion that could arise when Reduced motion is enabled.
+
+**Solution:** This warning is only displayed in development mode, but if you wish to disable it you can add the following line to your project's root file.
```

Comment:
```suggestion
**Solution:** Do nothing, this warning is safe to ignore as it is only displayed in development mode to avoid confusion. If you wish to disable it, you can add the following line to your project's root file:
```